### PR TITLE
add bootstrap-utilities.scss

### LINKED ISSUE #29026: Let's add Bootstrap-utilities.css to dist!
Hello, Bootstrap Community!

I think that Bootstrap would benefit from having a distributable `bootstrap-utilities.css` file, just as there is a `bootstrap-grid.css` and `bootstrap-reboot.css` files alongside the full `bootstrap.css`. 

Such a file:
- would not require absolutely any JavaScript in order to work, 
- only requires the addition of one scss file to the scss directory, 
- is completely customizable by editing the `_variables.scss` and `_utilities.scss` files, 
- provides a wealth of features to developers without requiring anything else.

The following code compiles to a `bootstrap-utilities.css` file and minifies to about 50KB.

```scss
// bootstrap/scss/bootstrap-utilities.scss

/*!
 * Bootstrap v4.3.1 (https://getbootstrap.com/)
 * Copyright 2011-2019 The Bootstrap Authors
 * Copyright 2011-2019 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 */


// Configuration

@import "functions";
@import "variables";
@import "mixins";
@import "utilities";


// Utilities

@import "utilities/api";
@import "utilities/sizing";
@import "utilities/text";
@import "utilities/visibility";
```

Let's do this!!

Pull Request: #29027

### PR DESCRIPTION
**Pull request by julian-hecker**
Add a bootstrap-utilities.scss file.

**Demo**
https://deploy-preview-29027--twbs-bootstrap.netlify.com/docs/4.3/getting-started/contents/

**CSS**
https://deploy-preview-29027--twbs-bootstrap.netlify.com/docs/4.3/dist/css/bootstrap-utilities.css

**Minified CSS**
https://deploy-preview-29027--twbs-bootstrap.netlify.com/docs/4.3/dist/css/bootstrap-utilities.min.css

Fixes #29026

### CODE REVIEW

**Comment by MartijnCuppens** [comment]
This needs more documentation, probably here: https://getbootstrap.com/docs/4.3/getting-started/contents/#css-files

**Comment by julian-hecker** [comment]
I'm not quite sure of how I should go about to include bootstrap.scss in the build process, and allow it to get compiled.

**Comment by MartijnCuppens** [comment]
I've updated the buildscript in the package file which will generate the css file & mappings. @XhmikosR, could you verify if that looks kosher to you?

It looks like targeting the folder instead of each file was faster for `node-sass`, see https://github.com/twbs/bootstrap/pull/29027/commits/7f3963228bcbb504dbe6993ca9d2e6e95d9efdeb

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
...
     "css-minify": "npm-run-all --parallel css-minify-*",
-    "css-minify-main": "cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap.min.css dist/css/bootstrap.css && cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-grid.min.css dist/css/bootstrap-grid.css && cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-reboot.min.css dist/css/bootstrap-reboot.css",
+    "css-minify-main": "cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap.min.css dist/css/bootstrap.css && cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-grid.min.css dist/css/bootstrap-grid.css && cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-utilities.min.css dist/css/bootstrap-utilities.css && cleancss --level 1 --format breakWith=lf --source-map --source-map-inline-sources --output dist/css/bootstrap-reboot.min.css dist/css/bootstrap-reboot.css",
```
Comment (XhmikosR) [inline package.json]: Can you move this patch to your previous one?

**Comment by XhmikosR** [comment]
I didn't set up these, but yeah, this should be fine since only the files we want to be generated don't have an underscore.

**Comment by XhmikosR** [comment]
I need to double check the new dist files before we merge this.

**Review by XhmikosR — APPROVED** [review]

**Review by MartijnCuppens — APPROVED** [review]

**Comment by julian-hecker** [comment]
@XhmikosR is this integrated properly? On `master` branch, I can see bootstrap-utilities in the scss directory and mentioned in documentation, but not in the dist directory. What to do?

**Comment by mdo** [comment]
@julian-hecker #28657 is open to build the files and commit them to the project.

**Pull request merged**